lib/router/src/utils.ts

```diff
@@ -18,7 +18,7 @@ export const parsePath: (path: string | undefined) => StoryData = memoize(1000)(
     };
 
     if (path) {
-      const [, viewMode, refId, storyId] = path.toLowerCase().match(splitPathRegex) || [];
+      const [, viewMode, refId, storyId] = path.match(splitPathRegex) || [];
```

Comment:
Does this mean URLs can include uppercase chars? This seems oddly inconsistent with our storyIds

Reply:
Do you recon it's better to enforce lowercasing refId's everywhere?

Reply:
I think so, if it's not too complicated

Reply:
Like we could just make it a requirement from the beginning